[REPORT] Crash report 856272c8f26610830cfa05fb961a9c48

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.github.adamantcheese.chan.dev/com.github.adamantcheese.chan.StartActivity}: java.lang.IllegalArgumentException: Cannot use local thread loadable for prefetching!
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3447)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3594)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2146)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:7762)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1047)
Caused by: java.lang.IllegalArgumentException: Cannot use local thread loadable for prefetching!
	at com.github.adamantcheese.chan.core.cache.FileCacheV2.enqueueMediaPrefetchRequestBatch(FileCacheV2.kt:214)
	at com.github.adamantcheese.chan.core.presenter.ThreadPresenter.onChanLoaderData(ThreadPresenter.java:589)
	at com.github.adamantcheese.chan.core.site.loader.ChanThreadLoader.quickLoad(ChanThreadLoader.java:278)
	at com.github.adamantcheese.chan.core.presenter.ThreadPresenter.requestInitialData(ThreadPresenter.java:308)
	at com.github.adamantcheese.chan.ui.controller.ViewThreadController.loadThreadInternal(ViewThreadController.java:507)
	at com.github.adamantcheese.chan.ui.controller.ViewThreadController.loadThread(ViewThreadController.java:484)
	at com.github.adamantcheese.chan.ui.controller.ViewThreadController.loadThread(ViewThreadController.java:573)
	at com.github.adamantcheese.chan.ui.controller.ViewThreadController.onCreate(ViewThreadController.java:142)
	at com.github.adamantcheese.chan.controller.Controller.addChildController(Controller.java:139)
	at com.github.adamantcheese.chan.ui.controller.ThreadSlideController.setRightController(ThreadSlideController.java:172)
	at com.github.adamantcheese.chan.ui.controller.BrowseController.showThread(BrowseController.java:477)
	at com.github.adamantcheese.chan.StartActivity.restoreFromSavedState(StartActivity.java:233)
	at com.github.adamantcheese.chan.StartActivity.setupFromStateOrFreshLaunch(StartActivity.java:168)
	at com.github.adamantcheese.chan.StartActivity.onCreate(StartActivity.java:157)
	at android.app.Activity.performCreate(Activity.java:7981)
	at android.app.Activity.performCreate(Activity.java:7970)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1307)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3422)
	... 11 more

```

--- a/Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/presenter/ThreadPresenter.java
+++ b/Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/presenter/ThreadPresenter.java
@@ -563,7 +563,7 @@ public class ThreadPresenter
                 }
             }
 
-            if (ChanSettings.autoLoadThreadImages.get() && !loadable.isLocal()) {
+            if (ChanSettings.autoLoadThreadImages.get() && !loadable.isLocal() && !loadable.isDownloading()) {
                 List<PostImage> postImageList = new ArrayList<>(16);
                 cancelPrefetching();
 

